wdl/imp_sub.wdl

```diff
@@ -178,9 +182,12 @@ task post_imputation {
         time Rscript --no-save /tools/r_scripts/plot_INFO_and_AF_for_imputed_chrs.R ${base} ${ref_panel_freq} ${chr} ${base}_varID_AF_INFO_GROUP.txt
 
         # annotate, edit tags and index
-        [[ ${base} =~ (.*)_qcd_imputed ]]
-        batch=${dollar}{BASH_REMATCH[1]}
-        edits="AF:AF_$batch INFO:INFO_$batch CHIP:CHIP_$batch AC_Het:AC_Het_$batch AC_Hom:AC_Hom_$batch HWE:HWE_$batch NS:NS_$batch AR2: DR2: IMP:"
+        edits="AR2: DR2: IMP:"
+        if ${add_batch_suffix}; then
+            [[ ${base} =~ (.*)_qcd_imputed ]]
+            batch=${dollar}{BASH_REMATCH[1]}
+            edits="AF:AF_$batch INFO:INFO_$batch CHIP:CHIP_$batch AC_Het:AC_Het_$batch AC_Hom:AC_Hom_$batch HWE:HWE_$batch NS:NS_$batch AR2: DR2: IMP:"
```

Comment:
need to also conditionally do the java -jar /tools/tageditin jadidaa below..... maybe have tageditcmd="" and assing the edit command which you then add below in to the place of the java call -> just a pass through in case no editing needed....

Reply:
I figured that the AR, DR2 and IMP tags are removed when they are (by default now) set like that for the tagediting. That's why I didn't just pass through if no batch suffix flag is false.